core/server/api/posts.js

```diff
@@ -105,7 +105,7 @@ posts = {
 
         // Push all of our tasks into a `tasks` array in the correct order
         tasks = [
-            apiUtils.validate(docName, {attrs: attrs, opts: options.opts || []}),
+            apiUtils.validate(docName, {attrs: attrs, opts: options.opts || ['formats']}),
```

Comment:
Hmmmm i am not sure why we use `options.opts`. It's only used in the post API, no where else.
I assume it's only used in tests to extend the possible options to pass in, but not sure that makes any sense.

Reply:
Oh it's used from the scheduler API. Will add a comment.